client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -26,11 +26,36 @@ const INSTALL_STATE_COMPLETE = 1;
 const INSTALL_STATE_INCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
-	componentDidUpdate( prevProps ) {
-		if ( prevProps.progressComplete < 100 && this.props.progressComplete >= 100 ) {
-			this.props.recordTracksEvent( 'calypso_plans_autoconfig_success', {
+	tracksEventSent = false;
+
+	recordAutoconfigTracksEvent( eventName, options = {} ) {
+		if ( ! this.tracksEventSent ) {
+			this.tracksEventSent = true;
+			this.props.recordTracksEvent( eventName, {
 				checklist_name: 'jetpack',
 				location: 'JetpackChecklist',
+				...options,
+			} );
+		}
+	}
+
+	componentDidUpdate( prevProps ) {
+		const { progressComplete, site } = this.props;
+
+		if ( prevProps.progressComplete < 100 && progressComplete >= 100 ) {
+			this.recordAutoconfigTracksEvent( 'calypso_plans_autoconfig_success' );
```

Comment:
I'm never seeing this fired. Should I assume that would be fixed by #33377?

Reply:
Correct, I'll rebase and should be fixed then.